content/pages/api-versioning-strategy/index.md

```diff
@@ -0,0 +1,231 @@
+
+If there is a significant change to API behavior, the new endpoint will be exposed under the new URL.
+For example, `https://upload.box.com/api/2.0/files/content` supports a multipart content type when uploading files to Box. If the new version of this API stops supporting this content type, it will be released under a new URL `https://upload.box.com/api/3.0/files/content`.
+
+## Release schedule and naming convention
+
+Box can introduce a new breaking change to certain endpoints **once per year**. An endpoint encompasses all supported HTTP methods within a specific API root. For example, the Sign Request endpoints include:
+
+| Method | Request URL                                        | Description                              |
+| ------ | -------------------------------------------------- | ---------------------------------------- |
+| GET    | `https://api.box.com/2.0/sign_requests/:id`        | Retrieves specific sign request details. |
+| GET    | `https://api.box.com/2.0/sign_requests/`           | Retrieves all sign requests.             |
+| POST   | `https://api.box.com/2.0/sign_requests/`           | Creates new sign requests.               |
+| POST   | `https://api.box.com/2.0/sign_requests/:id/resend` | Sends a specific sign request again.     |
+| POST   | `https://api.box.com/2.0/sign_requests/:id/cancel` | Cancels a specific sign request.         |
+
+Introducing a new version of the sign request endpoint means that all paths and HTTP methods listed above will support it. You can find comprehensive information about the endpoint version in the [Box API Reference](https://developer.box.com/reference/).
+
+### Naming convention
+
+New API versions are labeled according to the calendar year of their release.
+For example, if a new version of the Sign Requests endpoint is released in 2025, it will be named `2025.0`.
+
+Box can issue a new breaking change to API endpoints once per year, reserving the right to release an additional breaking change to address security or privacy concerns. In such cases, the new version will be incremented by one in the suffix.
+For example, if security issues need addressing in the previously released version `2025.0` of Sign Requests, the new version will be labeled `2025.1`.
+
+Each stable version is supported for a minimum of 12 months. This means that when a new version is released, the previous version becomes deprecated and will be available for usage, but no new features will be added.
+
+We strongly recommend updating your apps to make requests to the latest stable API version. However, if your app uses a stable version that is no longer supported, then you will get a response with an HTTP error code `404 - Not Found`. For details, see [Versioning Errors](#versioning-errors-and-fallbacks).
```

Comment:
The anchor `#versioning-errors-and-fallbacks` is incorrect. Looks like the subheader is called `fallback` and this anchor link should be `#versioning-errors-and-fallback`